queue.c

```diff
@@ -48,6 +48,11 @@
  * correct privileged Vs unprivileged linkage and placement. */
 #undef MPU_WRAPPERS_INCLUDED_FROM_API_FILE /*lint !e961 !e750 !e9021. */
 
+/* Code below here allows infinite loop controlling, especially for the infinite loop
+ * in idle task function (for example when performing unit tests). */
+#ifndef INFINITE_LOOP
```

Comment:
Can we move this default definition to [FreeRTOS.h](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/include/FreeRTOS.h#L2802) to avoid repeating in every source?

Reply:
Add configCONTROL_INFINITE_LOOP in FreeRTOS.h.